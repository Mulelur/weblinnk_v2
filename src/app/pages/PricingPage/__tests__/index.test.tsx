import * as React from 'react';
import { PricingPage } from '..';
import { MemoryRouter } from 'react-router-dom';
import { ThemeProvider } from 'styled-components';
import { themes } from 'styles/theme/themes';
import { HelmetProvider } from 'react-helmet-async';
import renderer from 'react-test-renderer';

const renderPage = () =>
  renderer.create(
    <MemoryRouter>
      <ThemeProvider theme={themes.light}>
        <HelmetProvider>
          <PricingPage />
        </HelmetProvider>
      </ThemeProvider>
    </MemoryRouter>,
  );

describe('<PricingPage />', () => {
  it('should match snapshot', () => {
    const notFoundPage = renderPage();
    expect(notFoundPage.toJSON()).toMatchSnapshot();
  });
});
