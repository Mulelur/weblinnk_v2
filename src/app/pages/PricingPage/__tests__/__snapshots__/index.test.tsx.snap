// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PricingPage /> should match snapshot 1`] = `
Array [
  .c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4 {
  padding: 0.25rem 1rem;
  font-size: 0.875rem;
  font-weight: 500;
  border-radius: 0.25rem;
  border: 1px solid rgba(58,52,51,0.05);
  background-color: rgba(255,255,255,1);
  background-color: transparent;
  color: rgba(58,52,51,1);
  border: none;
}

.c4:focus {
  outline: none;
  border: 1px solid rgba(215,113,88,1);
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-right: -1rem;
  color: rgba(215,113,88,1);
}

.c6 {
  color: rgba(215,113,88,1);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0.15rem 0.3rem;
  font-size: 0.875rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6:hover {
  opacity: 0.8;
}

.c6:active {
  opacity: 0.4;
}

.c6 .icon {
  margin-right: 0.2rem;
}

.c1 {
  display: none;
}

.c2 {
  color: rgba(215,113,88,1);
  padding-right: 0.8rem;
}

.c0 {
  box-shadow: 0 1px 0 0 rgba(58,52,51,0.05);
  height: 4rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  background-color: rgba(255,255,255,1);
  z-index: 2;
  width: 100%;
  padding-left: 10rem;
  padding-right: 10rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media only screen and (max-width:768px) {
  .c1 {
    display: block;
  }
}

@supports (-webkit-backdrop-filter:blur(10px)) or (backdrop-filter:blur(10px)) {
  .c0 {
    -webkit-backdrop-filter: blur(10px);
    backdrop-filter: blur(10px);
    background-color: rgba(255,255,255,0.75);
  }
}

@media only screen and (max-width:600px) {
  .c0 {
    padding-left: 1rem;
    padding-right: 3rem;
  }
}

@media only screen and (min-width:600px) {
  .c0 {
    padding-left: 5rem;
    padding-right: 5rem;
  }
}

@media only screen and (min-width:768px) {
  .c0 {
    padding-left: 17rem;
    padding-right: 7rem;
  }
}

@media only screen and (min-width:992px) {
  .c0 {
    padding-left: 18rem;
    padding-right: 8rem;
  }
}

<div
    className="c0"
    onClick={[Function]}
  >
    <div
      className="c1"
    >
      <div
        className=""
      >
        <div
          className="c2"
        >
          <svg>
            menu-icon.svg
          </svg>
        </div>
      </div>
    </div>
    <div
      className="c3"
    >
      <input
        className="c4"
        placeholder="search"
      />
    </div>
    <div
      className="c5"
    >
      <a
        className="c6"
        href="/profile"
        rel="noopener noreferrer"
        target="_blank"
        title="Documentation Page"
      >
        <svg>
          account-icon.svg
        </svg>
      </a>
      <a
        className="c6"
        href="https://github.com/react-boilerplate/react-boilerplate-cra-template"
        rel="noopener noreferrer"
        target="_blank"
        title="Github Page"
      >
        <svg>
          bell-icon.svg
        </svg>
      </a>
    </div>
  </div>,
  .c1 {
  color: rgba(58,52,51,1);
  font-size: 1.25rem;
}

.c2 {
  font-size: 0.8rem;
  line-height: 1.5;
  color: rgba(58,52,51,0.7);
  margin: 0.225rem 0 1.5rem 0;
}

.c8 {
  background-color: rgba(215,113,88,1);
  border-radius: 0.65rem;
  color: #fff;
  border: none;
  width: 100%;
  padding: 0.9rem;
}

.c4 {
  font-size: 0.8rem;
  line-height: 1.5;
  color: rgba(58,52,51,0.7);
  margin: 0.12rem 0 1rem 0;
}

.c3 {
  width: 20rem;
  height: 27rem;
  background-color: rgba(255,255,255,1);
  padding: 1.4rem;
  border-radius: 1rem;
  border: 1px solid rgba(215,113,88,1);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-shadow: 0px 12px 24px rgba(44,39,56,0.04),0px 24px 48px rgba(44,39,56,0.08);
}

.c5 {
  margin: 0;
  padding: 0;
  margin-bottom: 0.6rem;
}

.c6 {
  list-style: none;
  padding: 0.5rem 2.5rem;
  position: relative;
}

.c6::after {
  color: #fff;
  content: 'O';
  width: 2rem;
  height: 2rem;
  border-radius: 50%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: rgba(215,113,88,1);
  position: absolute;
  top: 5px;
  left: 0;
}

.c7 {
  border: 1px dashed #efefef;
  margin-bottom: auto;
}

.c0 {
  height: calc(100vh - 4rem);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media only screen and (max-width:768px) {

}

@supports (-webkit-backdrop-filter:blur(10px)) or (backdrop-filter:blur(10px)) {

}

@media only screen and (max-width:600px) {

}

@media only screen and (min-width:600px) {

}

@media only screen and (min-width:768px) {

}

@media only screen and (min-width:992px) {

}

<div
    className="c0"
  >
    <h1
      className="c1"
    >
      Get The Right Plan For You
    </h1>
    <p
      className="c2"
    >
      All plans including 14-days free trial from us
    </p>
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className="c3"
        >
          <h1
            className="c1"
          >
            Guardian
          </h1>
          <p
            className="c4"
          >
            What You'll Get
          </p>
          <ul
            className="c5"
          >
            <li
              className="c6"
            >
              <p
                className="c4"
              >
                lorem lepsum
              </p>
            </li>
            <li
              className="c6"
            >
              <p
                className="c4"
              >
                lorem lepsum ol rome
              </p>
            </li>
          </ul>
          <span
            className="c7"
          />
          <h4
            className=""
          >
            Free Forever
            <span
              className=""
            >
              
            </span>
          </h4>
          <button
            className="c8"
          >
            Choose
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
